ebmlite/examples.py

```diff
@@ -30,18 +28,18 @@
                                                         # into a string
 
 # Save xml
-with open(xmlFile1, 'wt') as f:
+with open(xmlFile1, b'wt') as f:
```

Comment:
These bytestring prefixes should be removed from anywhere they aren't necessary (e.g. this and filenames).